spec/swagger_helper.rb

```diff
@@ -0,0 +1,256 @@
+              public_reactions_count: { type: :integer, format: :int32 },
+              created_at: { type: :string, format: "date-time" },
+              edited_at: { type: :string, format: "date-time", nullable: true },
+              crossposted_at: { type: :string, format: "date-time", nullable: true },
+              published_at: { type: :string, format: "date-time" },
+              last_comment_at: { type: :string, format: "date-time" },
+              published_timestamp: { description: "Crossposting or published date time", type: :string,
+                                     format: "date-time" },
+              reading_time_minutes: { description: "Reading time, in minutes", type: :integer, format: :int32 },
+              user: { "$ref": "#/components/schemas/SharedUser" },
+              flare_tag: { "$ref": "#/components/schemas/ArticleFlareTag" },
+              organization: { "$ref": "#/components/schemas/SharedOrganization" }
+            },
+            required: %w[type_of id title description cover_image readable_publish_date
+                         social_image tag_list tags slug path url canonical_url comments_count
+                         positive_reactions_count public_reactions_count created_at edited_at
+                         crossposted_at published_at last_comment_at published_timestamp user
+                         reading_time_minutes]
+          },
+          SharedUser: {
+            description: "The resource creator",
+            type: "object",
+            properties: {
+              name: { type: :string },
+              username: { type: :string },
+              twitter_username: { type: :string, nullable: true },
+              github_username: { type: :string, nullable: true },
+              website_url: { type: :string, format: :url, nullable: true },
+              profile_image: { description: "Profile image (640x640)", type: :string },
+              profile_image_90: { description: "Profile image (90x90)", type: :string }
+            }
+          },
+          SharedOrganization: {
+            description: "The organization the resource belongs to",
+            type: "object",
+            properties: {
+              name: { type: :string },
+              username: { type: :string },
+              slug: { type: :string },
+              profile_image: { description: "Profile image (640x640)", type: :string, format: :url },
+              profile_image_90: { description: "Profile image (90x90)", type: :string, format: :url }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  # Specify the format of the output Swagger file when running "rswag:specs:swaggerize".
+  # The swagger_docs configuration option has the filename including format in
+  # the key, this may want to be changed to avoid putting yaml in json files.
+  # Defaults to json. Accepts ":json" and ":yaml".
+  config.swagger_format = :json
```

Comment:
I've switched the format of the outputted documentation from `yaml` to `json`, but after reading https://oai.github.io/Documentation/specification-structure.html I'm not sure which is best